packages/wellbeing-mock-data/wellbeing-mock-data.js

```diff
@@ -130,20 +130,106 @@ function createMockActivities() {
   }
 }
 
+
+function getRandomHomeButExcludeCurrent(current_id) {
+  var homeIds = Homes.find().map(function (e) { return e._id; });
```

Comment:
Rather than mapping over all homes to remove a specific home, why not write a MongoDB query that excludes the home? It would be more succinct and declarative (easier to read and understand).

The MongoDB [$not](https://docs.mongodb.com/manual/reference/operator/query/not/) operator can be used here to exclude the home document.

Reply:
Good point. I'll change this.